src/main.c

```diff
@@ -1,26 +1,11 @@
 #include <stdlib.h>
-#include <time.h>
 
-#include "../include/raylib.h"
-#include "../include/raymath.h"
+#include "graphics/gui.h"
 
+int main(int argc, char **argv) {
 
-int screenWidth = 800;
-int screenHeight = 480;
-char title[] = "Raylib Template";
+    if (argc == 1) runGameGui();
```

Comment:
Perhaps replace the 1 with a constant from a constant file.

Reply:
I decided to leave it a literal since there isn't a suitable constant we can use since we're checking the number of command line arguments.